share/functions/__fish_print_hostnames.fish

```diff
@@ -62,10 +62,11 @@ function __fish_print_hostnames -d "Print a list of known hostnames"
                 | string trim | string replace -r -a '\s+' ' ')
             end
 
+            cd $relative_path
```

Comment:
I'll switch it to `builtin cd`. As for people writing their own cd functions if they do so in a fashion that breaks a usage like this one they've done the equivalent of executing the [HCF](https://www.techopedia.com/definition/19720/halt-and-catch-fire-hcf) opcode.